pym/bob/cmds/build.py

```diff
@@ -833,6 +869,8 @@ def commonBuildDevelop(parser, argv, bobRoot, develop):
         else:
             parser.error("Malformed define: "+define)
 
+    startTime = time.time()
```

Comment:
Actually it does include all relevant time here. The parsing (`recipes.parse()`) and package calculation (`recipes.generatePackages(...)`) are done below that line...